docs/index.md

```diff
@@ -3,17 +3,17 @@
   title: ""
 ---
 
-# AddressBook Level-3
+# BayMeds v.2103
 
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
-[![codecov](https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg)](https://codecov.io/gh/se-edu/addressbook-level3)
+[![CI Status](https://github.com/AY2324S1-CS2103T-T15-2/tp/actions/workflows/gradle.yml/badge.svg)](https://github.com/AY2324S1-CS2103T-T15-2/tp/actions)
+[![codecov](https://codecov.io/gh/AY2324S1-CS2103T-T15-2/tp/branch/master/graph/badge.svg)](https://codecov.io/gh/AY2324S1-CS2103T-T15-2/tp)
 
 ![Ui](images/Ui.png)
 
-**AddressBook is a desktop application for managing your contact details.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
+**BayMeds v.2103 is a desktop application for managing your medications.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
```

Comment:
Would v.2103 be confusing for readers? Should we stick to a single name (BayMeds)?